hub/executors/encoders/image/onnx_image_encoder/Dockerfile

```
@@ -0,0 +1,9 @@
+FROM onnx/onnx-base
+
+ADD jina .src
+
+RUN pip install .src/.
```

No need to do that anymore, previously it was like this because jina was not released and not public accessible. Now you can simply do

```suggestion
RUN pip install jina
```